parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -253,8 +258,15 @@ pub mod pallet {
 			// Reward relayer from the sovereign account of the destination parachain
 			// Expected to fail if sovereign account has no funds
 			let sovereign_account = sibling_sovereign_account::<T>(channel.para_id);
-			let delivery_cost = Self::get();
-			T::Token::transfer(&sovereign_account, &who, delivery_cost, Preservation::Preserve)?;
+			let delivery_cost = Self::calculate_delivery_cost(message.encode().len() as u32);
+			if delivery_cost > T::Token::minimum_balance() {
```

Comment:
This seems unnecessary. In practice `delivery_cost` is always going to be greater than the minimum balance.

Reply:
So will that always be true in other networks? Notice the `ED` in Polkadot is pretty high.
https://github.com/polkadot-fellows/runtimes/blob/30e0dbfdcb78722ed61325c0ebf1efdcdb6033ba/relay/polkadot/constants/src/lib.rs#L28

Reply:
Still seems unnecessary. The relayer would have already have had to fund their account with an amount greater than `ED` in order to submit the message.